### ISSUE TITLE
Issue with renaming media while uploading 

### ISSUE THREAD

**Issue by kjnanda** [rtCamp/rtMedia#1640]
Renaming the media takes the extension of `mp4` or `mp3`! However, once the media is upload it functions properly. 

@SID177 Please check this below visual. 

https://drive.google.com/file/d/1F3BeA9Cszi25uE5H5AlMnYEWDrKZDRrN/view

**Comment by pooja-muchandikar** [comment]
@SID177 

Cross-checked the issue with following test environment and works as expected.

```
rtMedia: v4.6.3 ( pre-release branch )
BuddyPress: v6.0.0 ( nouvea and legacy )
Theme: 2020, 2019
```

**Comment by pooja-muchandikar** [comment]
Tested with `develop` branch and the fix works as expected.

**Comment by pooja-muchandikar** [comment]
Tested with `master` branch and the fix works as expected.

**Comment by pooja-muchandikar** [comment]
Released with`v4.6.4`
